commands/builds/create.js

```diff
@@ -8,64 +8,58 @@ let uuid = require('node-uuid');
 let os = require('os');
 let path = require('path');
 let request = require('request');
+let exec = require('child_process').execSync
 
-function uploadCwdToSource(app, cwd, fn) {
+function checkTarInstall(tar) {
+  let tarVersion = exec(tar+" --version").toString()
+
+  if (!tarVersion.match(/GNU tar/)) {
+    throw  `This command requires GNU Tar to compress your app code.
```

Comment:
It does appear to work sometimes (I've had it work with sample apps).
I don't mind turning it to a warning.